recipes/meta/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "meta" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "a35dd5bc3ccffaaf83aa488233467ae85dc5374aea3068c34a6dc85ea29e27a9" %}
+{% set build_variant = "osmesa" if linux else "qt" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bagari/meta/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('meta', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [not win]
+
+  host:
+    - python >=3.9,<3.10
+    - pip
+    - boost =1.82
+    - itk =5.3.0
+    - libitk =5.3.0
+    - libitk-devel =5.3.0
+    - hdf5 >=1.14.2,<1.14.3.0a0
+    - vtk-base >=9.2.6,<9.3.0 build=*{{ build_variant }}*
+    - libexpat >=2.5.0,<2.6.0
+    - vtk-io-ffmpeg >=9.2.6,<9.3.0 build=*{{ build_variant }}*
+
+  run:
+    - qhull
+    - boost =1.82
+    - itk =5.3.0
+    - libitk =5.3.0
+    - libitk-devel =5.3.0
+    - hdf5 >=1.14.2,<1.14.3.0a0
+    - vtk-base >=9.2.6,<9.3.0 build=*{{ build_variant }}*
```

Comment:
for packages that appear in the `host` section, we prefer to rely their `run_exports`. Sometimes they only require a subset of the package that avoids compilation time requirements.

Reply:
I updated the run section based on your suggestions. Thanks